draft-regext-rdap-extensions.md

```diff
@@ -175,6 +175,47 @@ may also be an extension for a completely separate purpose. Only consultation
 of the definition of `fizzbuzz_1` will determine its relationship with
 `fizzbuzz_0`. Additionally, `fizzbuzz_99` may be the predecessor of `fizzbuzz_0`.
 
+## Backwards-Compatible Changes
+
+If an RDAP extension author wants to publish a new version of an
+extension that is backwards-compatible with the previous version, then
+one option is for the new version of the extension to define a new
+identifier, as well as requiring that both the previous conformance
+code and the new conformance code be included in responses.  That way,
+clients relying on the previous version of the extension will continue
+to function as intended, while clients wanting to make use of the
+newer version of the extension can check for the new conformance code
+in the response.
+
+This approach can be used for an arbitrary number of new
+backwards-compatible versions, with each successor version requiring
+the inclusion of all of the conformance codes required by the previous
+version.
```

Comment:
"arbitrary number" threw me off a bit. Should we further clarify how many previous backwards-compatible versions need to be included in the rdapConformance array? Or, is it better to leave this decision to the RDAP server operators?

Reply:
Thanks, I think the latter, updated text accordingly.